validator/keymanager/imported/keymanager.go

```diff
@@ -66,8 +67,9 @@ type AccountStore struct {
 // DefaultKeymanagerOpts for a imported keymanager implementation.
 func DefaultKeymanagerOpts() *KeymanagerOpts {
 	return &KeymanagerOpts{
-		EIPVersion: eipVersion,
-		Version:    "2",
+		EIPVersion:         eipVersion,
+		Version:            "2",
+		DisabledPublicKeys: [][]byte{},
```

Comment:
hmm, this actually won't work :( because the `WalletResponse` pb  requires all config to be `map<string, string>`.
I can:
1) update the pb to allow [][]byte
2) transform DisabledPublicKeys to be a string comma delimited